packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -331,8 +339,8 @@ export async function ensureAvailableUsers(
     busyTimesFromLimitsBookingsAllUsers = await getBusyTimesForLimitChecks({
       userIds: eventType.users.map((u) => u.id),
       eventTypeId: eventType.id,
-      startDate: input.dateFrom,
-      endDate: input.dateTo,
+      startDate: startTime.format(),
+      endDate: endTime.format(),
```

Comment:
Need to do more testing but this could be potentially why limits checks have been weird/buggy. This value was being sent in a local date format

Reply:
`dayjs().format()` can also be local tz.

Reply:
Unless we really need a specific tz format. It's generally better to use `dayjs.utc()`. It is explicit and anyone working on it can instantly get the tz the date is in.

Reply:
This is now using .utc()